query-builder/src/main/java/com/datastax/oss/driver/internal/querybuilder/update/DefaultUpdate.java

```diff
@@ -154,6 +180,15 @@ public String asCql() {
       }
     }
 
+    if (ttl != null) {
+      builder.append(" USING TTL ");
```

Comment:
Same here, we need to check if both clauses are present and append an `AND` if required.

Reply:
Can't we extract that logic somewhere? (superclass)?

Reply:
Again, not really worth it, this would "save" like 4 lines of code.

Reply:
My worry is not duplication but the problem of remembering to add that specific part of the code to every implementation that is doing insert/modification.
Maybe `usingTtl` method should be created in the `OngoingAssignment` interface?